### PR TITLE
Completed project requirements

### PR DESCRIPTION
**Pull request by BucurEva87**
- created the redux store based on the pieces of information from the COVID API
- created the main page to display global information about the countries
- created the details page to show country-specific information regarding the infections and death rates
- tested components

### CODE REVIEW

File: src/styles/index.scss [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+}
+
+a:link,
+a:visited,
+a:active,
+a:hover {
+  color: $linkColor;
+  text-decoration: none;
+  text-transform: uppercase;
+}
```
Comment (codecaiine) [inline src/styles/index.scss:28]: - [ ] Your project is good :clap: But there are some parts from  [design guidelines](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)) you forgot :cry:  Please make sure to follow the design including Typography (font face, size, and weight) and Layout (composition and space between elements) as required for this project.  As a developer, it is very important to follow the original design. Consider the case where a client/company gives you a design, you need to provide and reproduce it exact product he wants in order to make it approve your work. In your future job, you will judge your ability to satisfy your clients or company. Please make it work :+1: 
- [ ]  For this project we expect you to create a mobile web application. Please make sure that your design is responsive and adapts to mobile resolutions. :smile: 
Please check the screenshots below to see the differences :arrow_down: 

#### Your design Home page
Screenshot from 2022-10-22 20-30-25

#### Your design Details page
Screenshot from 2022-10-22 20-30-30

#### Your Expected Design Home page
Screenshot from 2022-10-22 20-32-39

#### Your expected design Details page
Screenshot from 2022-10-22 20-32-58



- [ ] Please make sure that you have your home and details page following the design guidelines and also the filter as defined in the design below :arrow_down: As a developer, it is very important to pay attention to details.  Kindly change the color theme that you chose for your design and kindly use one of the two provided color theme alternatives. This way, your project will match the Behance design and your project will be ready for approval. :smile: 

#### You can check the demo below :point_down: as an example
https://covid19-metrics-app-aka.netlify.app/

File: src/components/Country/Country.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+const Country = ({ name, flag }) => (
+  <article>
+    <h2>
+      <Link to={name}>{name}</Link>
+    </h2>
+    <img src={flag} alt="Country flag not available" className={style.flag} />
+  </article>
+);
+
```
Comment (codecaiine) [inline src/components/Country/Country.js:14]: - [ ] Good job but :+1:  but please make sure that on your home page we have a list of categories that could be filtered using given parameters, e.g. category name as required for this project. You can see check the screenshots below to  see an implementation :arrow_down: 

### Filter implementation
Screenshot from 2022-10-22 20-41-01
Screenshot from 2022-10-22 20-41-09

File: README.md [hunk 3]
```diff
@@ -58,6 +58,10 @@ Contributions, issues, and feature requests are welcome!
 
 Feel free to check the [issues page](../../issues/).
 
+## Aknowledgements
+
+Hats off to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) for providing the design for this project!
+
```
Comment (codecaiine) [inline README.md:68]: - [ ] Great job but I can not find a presentation video of your project. Please record a maximum 5 mins video presenting the features of the project you built as required for this project +1
196996460-fe0a77f2-d190-40f3-86a9-d1945102cc73

**Review by codecaiine — CHANGES_REQUESTED** [review]
Hi @BucurEva87 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- Project is impressive :heavy_check_mark: 
- Great implementation :heavy_check_mark: 
- No linters errors :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/styles/index.scss (hunk 1 above)
Comment (BucurEva87) [reply src/styles/index.scss:28]: @codecaiine , thank you for your kind review and for your keen suggestions, but I can't help but be disappointed at the idea that [this project](https://tourmaline-syrniki-7038a4.netlify.app/) passed without matching a single colour or style, while mine, even if it respects the design much better, is not good enough. I also see no filter feature in that project. I know that comparing is not an excuse, but thank you for making me fail the entire Microverse program - for this is the second time I'll have to repeat the week (and, so, I'll be thrown out of the program)! It's not like I can do anything in about 105 minutes.

**Pull request closed without merging**